guides/user/canary/stage/index.md

```diff
@@ -0,0 +1,183 @@
+
+   * For __Analysis Type__, select __Growing__ or __Sliding Lookback__.
+
+     - In a growing analysis, a judgment is taken every [interval] minutes, but
+     each judgment goes all the way back to the beginning of the __Lifetime__.
+
+     - A sliding Lookback also makes a judgment every [interval], but each
+     judgment only looks at the data from the most recent lookback duration.
+     It would not be unusual for the __Interval__ and the __look-back__
+     duration to be the same, but they don't have to be.
+
+   ![Canary stage declaration](/guides/user/canary/stage/stage_config_analysis.png)
+
+1. Describe the metric scope.
+
+   > You can enter server groups and regions by name, but you can also click the
+   > magic wand here to automatically populate the fields with expressions that
+   > resolve to available resources. Those resources are a starting point, which
+   > you can edit to match the specific resources you need.
+
+   * __Baseline__
+
+     The server group to treat as the "control" in the canary
+     analysis&mdash;that is, the deployment against which to compare the canary
+     deployment.
+
+   * __Baseline Region__
+
+     The region in which the baseline server group is deployed.
+
+   * __Canary__
+
+     The server group to treat as the experiment in the analysis.
+
+   * __Canary Region__
+
+     The region in which that canary server group is deployed.
+
+   * __Step__
+
+     The interval, in seconds, for the metric time series.
+
+
+   * __Start Time__ and __End Time__ (for retrospective)
+
+     For a retrospective analysis, the specific time span over which to conduct
+     the analysis.
+
+   * <a name="extended-params" />__Extended Params__
+
+     Add any additional parameters, which are specific to the metric sources and
+     which can be used to refine the scope of the analysis. These parameters can
+     provide variable bindings for use in the expansion of custom filter
+     templates [specified in the canary
+     config](/guides/user/canary/config/filter_templates/).
+
+   ![Canary stage declaration](/guides/user/canary/stage/metric_scope.png)
+
+1. Adjust the __Scoring Thresholds__, if needed.
+
+   The thresholds are pre-populated based on those configured in the main
+   [canary config](/guides/user/canary/config/), but you can override them here.
+
+1. Specify the accounts you're using for metrics and storage.
+
+   * The __Metrics Account__ points to the telemetry service provider account
+   you configured [here]().
+
+   * The __Storage Account__ points to the GCS or S3 account you configured
+   [here]().
+
+   ![Canary stage declaration](/guides/user/canary/stage/advanced_settings.png)
+
+   ## Time scope in the canary stage
+
+   When you define your canary stage, you have a lot of choices about how to
+   manage the analysis over time, beginning with the analysis type:
+
+   * real time
+
+   * retrospective
+
+   So let's think about it in those two categories...
```

Comment:
Would strip these out for now if they're not done.

Reply:
Ah, I don't even know why those're still there. I think they're explained enough in the procedure above.